sdk/textanalytics/ai-text-analytics/test/apiKey.spec.ts

```diff
@@ -77,4 +77,449 @@ describe("[API Key] TextAnalyticsClient", function() {
     assert.equal(results.length, 1);
     assertAllSuccess(results);
   });
+
+  describe("#health", () => {
+    it("input strings", async () => {
+      const poller = await client.beginAnalyzeHealthcare([
+        "Patient does not suffer from high blood pressure.",
+        "Prescribed 100mg ibuprofen, taken twice daily."
+      ]);
+      const result = await poller.pollUntilDone();
+      for await (const doc of result) {
+        if (!doc.error) {
+          assert.ok(doc.id);
+          assert.ok(doc.statistics);
+          assert.ok(doc.entities);
+          assert.ok(doc.relations);
+        }
+      }
+    }).timeout(1000000);
```

Comment:
this timeout should be a const, but it's also so big as to be nearly ludicrous. Certainly it shouldn't be this large during playback, at least?

Reply:
I hoisted it to the `describe` block conditioned by being in a live or record mode.